europe/README.EUROPE

```diff
@@ -106,6 +106,28 @@ The Malin Head datum is used in the Republic of Ireland.
 
 * OSGM15_Malin.gtx
 
+### Netherlands: RD and NAP -> ETRS89
+
+*Source*: [NSGI](https://www.nsgi.nl/geodetische-infrastructuur/producten/coordinatentransformatie)  
+*Format*: NTv2 & GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)  
+
+Variant 1:
+
+* rdcorr2018.gsb
+* nlgeo2018.gtx
+
+Recommended variant, applies the datum transformation as a separate step using
+a 3D similarity transformation.
+
+Variant 2:
+
+* rdtrans2018.gsb
+* naptrans2018.gtx
```

Comment:
isn't there an inversion between nlgeo2018.gtx and naptrans2018.gtx ? I see EPSG has only entries for rdtrans2018.gsb + nlgeo2018.gtx ? Or is EPSG shipping with a mix of current variant 1 & variant 2

Reply:
Yes, unfortunately EPSG 9.8.6 is shipping a mix of variant 1 & 2, as mentioned in the OP.

I've included all 4 grids in this PR to not have to add them later once EPSG is updated, even though EPSG only uses a subset of the grids at time of writing.

Reply:
> Or is EPSG shipping with a mix of current variant 1 & variant 2

Ah my bad, you mentionned that in your initial message. From a meeting that I attended, I know EPSG is not a big fan of the oddities of RDNAPTRANS ;-)